app/src/index.css

```diff
@@ -27,7 +27,7 @@
   --color-white: #ffffff;
   --color-black: #000000;
 
-  --error-color: var(--color-red);
+  --error-color: var(--color-pale);
```

Comment:
Maybe we could use a more vibrant tone, this one is a little mellow so doesn't standout enough IMO

Reply:
Yeah I was playing around with this. I found the red difficult to see against the green background.

Reply:
Ah yes I hadn't thought of that, in the design it was only tested against the black background. Maybe the orange #D9532C would work nice?